src/App.css

```diff
@@ -0,0 +1,87 @@
+/* General Styles */
+* {
```

Comment:
Looks like you have some very large tabs in this file. Make sure the settings in VS code are 2 space tabs!